### PR TITLE
add support for lambda task

### PR DESCRIPTION
**Pull request by ziyunli**

### CODE REVIEW

File: test/test_DecisionResponse.js [hunk 1]
```diff
@@ -49,6 +49,15 @@ describe('DecisionResponse', function(){
...
     });
 
+    it('#schedule_lambda()', function() {
+      var dr = new DecisionResponse(swfClientMock, '12345', 'my-tasklist');
+      dr.schedule_lambda({
+        /*
+        id: 'my-lambda-id',
+        name: 'my-lambda-name'*/
+      });
+    });
```
Comment (ktonon) [inline test/test_DecisionResponse.js:59]: can this test fail?

Comment (ziyunli) [reply test/test_DecisionResponse.js:59]: I don't think so, but neither does the previous test.

Comment (ktonon) [reply test/test_DecisionResponse.js:59]: I guess its more documentation than a test?

**Review by ktonon — APPROVED** [review]

File: test/test_DecisionResponse.js (hunk 1 above)
Comment (dougmoscrop) [reply test/test_DecisionResponse.js:59]: I feel like you could use sinon or otherwise assert that it did 'something'.

**Pull request merged**